src/Discovery/DiscoveryDetails/DataDownloadList/DataDownloadList.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { List, Button } from 'antd';
+import { DownloadOutlined } from '@ant-design/icons';
+import DataDownloadListItem from './Utils/DataDownloadListItem';
+import './DataDownloadList.css';
+import ProcessData from './Utils/ProcessData';
+
+const DataDownloadList = ({ sourceFieldData }: any) => {
+  const data = ProcessData(sourceFieldData);
+  if (data.length === 0) {
+    return null;
+  }
+  return (
+    <div className='discovery-modal__data-download-list'>
+      <h3>Data Download Links</h3>
```

Comment:
we could remove this

Reply:
Removed header, and updated tests to not check for header.